roles/elastic-stack/ansible-elasticsearch/defaults/main.yml

```diff
@@ -2,7 +2,8 @@
 elasticsearch_cluster_name: wazuh
 elasticsearch_node_name: node-1
 elasticsearch_http_port: 9200
-elasticsearch_network_host: 127.0.0.1
+elasticsearch_network_host: 0.0.0.0
```

Comment:
I think we should set this default value as `127.0.0.1`

Reply:
Hi @manuasir ,

No problem. We set it to 0.0.0.0 to simulate the error reported in the related issue.

I've just changed its value to `127.0.0.1` in https://github.com/wazuh/wazuh-ansible/commit/125b2114e6c77f089aac16bf4668cde2290efcbc .

https://github.com/wazuh/wazuh-ansible/blob/125b2114e6c77f089aac16bf4668cde2290efcbc/roles/elastic-stack/ansible-elasticsearch/defaults/main.yml#L5 .

Kind regards,

Rshad